File: 0058-length-of-last-word/0058-length-of-last-word.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        # str = s.split(" ")[-1]
        # if(str == " "):
        #     strs = str[-1]
        #     return len(strs)
        str = s.split()
        return len(str[-1])

File: 0058-length-of-last-word/0058-length-of-last-word.c
int lengthOfLastWord(char* s) {
    int count = 0;
    for (int i = strlen(s) - 1; i >= 0; i--) {
        if (s[i] == ' ' && count > 0)
            break;
        if (s[i]!=' '){
            count++;
        }
    };
    return count;
}